Documentation/helm-operator.md

```diff
@@ -145,6 +146,8 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.rbdPodLabels`                  | Labels to add to the CSI RBD Pods.                                                                                          | <none>                                                    |
 | `csi.volumeReplication.enabled`     | Enable Volume Replication.                                                                                                  | `false`                                                   |
 | `csi.volumeReplication.image`       | Volume Replication Controller image.                                                                                        | `quay.io/csiaddons/volumereplication-operator:v0.1.0`     |
+| `csi.csiAddons.enabled`     | Enable CSIAddons                                                                                                  | `false`                                                   |
+| `csi.csiAddons.image`       | CSIAddons Sidecar image.                                                                                        | `quay.io/csiaddons/k8s-sidecar:v0.1.0`     |
```

Comment:
I am not sure if this will be fine. because our project itself is CSIAddons and just having addons maybe not suitable. in `csi.csiAddons.image` both csi and csiaddons are different. I would prefer it to be this way. Let me know if you still want me to change it. @BlaineEXE

Reply:
That seems fine. :+1: